layouts/partials/home.html

```diff
@@ -0,0 +1,69 @@
+<!-- Main jumbotron for a primary marketing message or call to action -->
+    <div class="container-fluid">
+        <div class="jumbotron banner">
+            <div class="container">
+                <div class="row">
+                    <h2>Welcome to NASA WorldWind</h2>
+                </div>
+                <div class="row">
+                    <div class="col-md-8">
+                    <p>WorldWind is a virtual globe technology providing the means to visualize, manipulate and analyze data in a real world virtual representation.</p>
```

Comment:
The alignment with the two text rows and the buttons is off. Also, as I look at this is a new layout, I think the second row of text may be superfluous. How would it look if we just had the set "Welcome to NASA WorldWind" or "NASA WorldWind", made the text larger, and used a color that stood out better against the background?

I realize this is a straight port from the site currently at https://worldwind.arc.nasa.gov, and may be an existing issue.